content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md

```diff
@@ -0,0 +1,116 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Executa o melhor esforço para reverter as alterações feitas no host por 'kubeadm init' ou 'kubeadm join'
+
+### Sinopse
+
+Executa o melhor esforço para reverter as alterações feitas no host por 'kubeadm init' ou 'kubeadm join'
+
+O comando "reset" executa as seguintes fases:
+
+
+```
+preflight           Run reset pre-flight checks
+remove-etcd-member  Remove a local etcd member.
+cleanup-node        Run cleanup node.
+```
+
+
+```
+kubeadm reset [flags]
+```
+
+### Opções
+
+   <table style="width: 100%; table-layout: fixed;">
+<colgroup>
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">--cert-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Padrão: "/etc/kubernetes/pki"</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>O caminho para o diretório onde os certificados estão armazenados. Se especificado, limpe este diretório</p></td>
+</tr>
+
+<tr>
+<td colspan="2">--cri-socket string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>Caminho para o socket CRI se conectar. Se o kubeadm vazio tentar detectar automaticamente esse valor; use essa opção somente se você tiver mais de um CRI instalado ou se tiver um socket CRI não padrão.</p></td>
```

Comment:
In the "Se kubeadm vazio..." sentence, "empty" refers to the value of the argument. Therefore, the sentence should be "Se vazio, o kubeadm tentará...".